packages/kotti-ui/source/kotti-comment/utilities.ts

```diff
@@ -2,4 +2,4 @@ import escape from 'lodash/escape'
 
 export const defaultParser = (message: string) => escape(message)
 export const defaultPostEscapeParser = (message: string) =>
-	message.replace(/\n/g, '</br>')
+	message.replace(/\n/gm, '</br>')
```

Comment:
However, upon seeing this thing again... why is this `</br>` and not either `<br />` or `<br>`?